From khali@linux-fr.org Sun Jan 22 03:28:59 2006
Date: Sun, 22 Jan 2006 12:28:57 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Andrew Morton <akpm@osdl.org>
Cc: Dave Jones <davej@redhat.com>, Greg KH <greg@kroah.com>, Ben Gardner <bgardner@wabtec.com>
Subject: I2C: fix sx200_acb build on other arches
Message-Id: <20060122122857.008a5e61.khali@linux-fr.org>


scx200_acb is only useful on a few Geode-based systems, and won't
compile on non-x86 systems due to the lack of asm/msr.h, as reported
by Andrew Morton. Thus, we should make that driver depend on X86.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
---
 drivers/i2c/busses/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/i2c/busses/Kconfig
+++ gregkh-2.6/drivers/i2c/busses/Kconfig
@@ -390,7 +390,7 @@ config SCx200_I2C_SDA
 
 config SCx200_ACB
 	tristate "Geode ACCESS.bus support"
-	depends on I2C && PCI
+	depends on X86_32 && I2C && PCI
 	help
 	  Enable the use of the ACCESS.bus controllers on the Geode SCx200 and
 	  SC1100 processors and the CS5535 and CS5536 Geode companion devices.
